packages/authorization-utils/src/msg.rs

```diff
@@ -93,6 +163,9 @@ pub enum ExecuteMsg {
     // Polytone callback listener
     #[serde(rename = "callback")]
     PolytoneCallback(CallbackMessage),
+    // Hyperlane callback listener
+    #[serde(rename = "handle")]
```

Comment:
now I get why in the authorizations contract execute match statement on the msg you had `HyperlaneCallback(*handle_msg*)` while `PolytoneCallback(*callback_msg*)`. feel free to disregard that one!